### PR TITLE
Dashboard fixes

### PR DESCRIPTION
**Pull request by w41eed**

### CODE REVIEW

File: src/Components/Dashboard/ResumeBank.js [hunk 1]
```diff
@@ -231,7 +209,7 @@ class JobBoard extends Component {
               ))}
             </Grid>
           </div>
-        </PerfectScrollbar>
+        {/* </PerfectScrollbar> */}
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/ResumeBank.js:212]: Is this meant to be commented out?

File: src/Components/Dashboard/ResumeBank.js [hunk 2]
```diff
@@ -32,8 +31,11 @@ const useStyles = makeStyles(() => ({
     alignItems: 'center',
   },
 
+  padding: {
+  },
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/ResumeBank.js:35]: empty CSS styling? intentional or not?

File: src/Components/Dashboard/Jobs.js [hunk 3]
```diff
@@ -258,7 +258,7 @@ class JobBoard extends Component {
            }
           </Grid>
         </div>
-      </PerfectScrollbar>
+      {/* </PerfectScrollbar> */}
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Jobs.js:261]: Meant to be removed?

File: src/Components/Dashboard/Home.js [hunk 4]
```diff
@@ -307,11 +314,12 @@ class Home extends Component {
...
                 </Grid>
               </Grid>
+              
           
             }
-          </Grid>
+          {/* </Grid> */}
       </div>
-      </PerfectScrollbar>
+      {/* </PerfectScrollbar> */}
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Home.js:322]: Meant to be commented out?

File: src/Components/Dashboard/Home.js [hunk 5]
```diff
@@ -232,20 +235,24 @@ class Home extends Component {
...
                 </Grid>
               </Grid>
+              
             :
               <Grid
                 container
-                item xs={12}
-                spacing={1}
+                // item xs={12}
+                // spacing={1}
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Home.js:243]: Probably can remove these.

File: src/Components/Dashboard/Home.js [hunk 6]
```diff
@@ -156,18 +156,21 @@ class Home extends Component {
...
             </Grid>
+            </Grid>
 
             {this.props.isSeniorExec ?
               <Grid
                 container
-                item xs={12}
-                spacing={1}
+                // item xs={12}
+                // spacing={1}
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Home.js:165]: Probably can remove these.

File: src/Components/Dashboard/Community.js [hunk 7]
```diff
@@ -218,7 +218,7 @@ class JobBoard extends Component {
               ))}
             </Grid>
           </div>
-        </PerfectScrollbar>
+        {/* </PerfectScrollbar> */}
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Community.js:221]: Meant to be removed?

File: src/Components/Dashboard/CoffeeChats.js [hunk 8]
```diff
@@ -229,7 +229,7 @@ class CoffeeChats extends Component {
               }
             </Grid>
           </div>
-        </PerfectScrollbar>
+        {/* </PerfectScrollbar> */}
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/CoffeeChats.js:232]: Meant to be removed?

**Review by ahmedhamodi — COMMENTED** [review]
Some minor adjustments. Also there are some merge conflicts needing to be addressed!

File: src/Components/Dashboard/ResumeBank.js (hunk 1 above)
Comment (w41eed) [reply src/Components/Dashboard/ResumeBank.js:212]: yeah, i'm gonna leave it commented out for now. need to test something with that but i'll do that later

**Review by ahmedhamodi — APPROVED** [review]
Great work! Thanks for doing this.

**Pull request merged**